examples/ec2-autoscaling/main.tf

```diff
@@ -120,6 +120,18 @@ module "ecs_service" {
 
       # Example image used requires access to write to root filesystem
       readonly_root_filesystem = false
+
+      enable_cloudwatch_logging              = true
+      create_cloudwatch_log_group            = true
+      cloudwatch_log_group_name              = "/aws/ecs/${local.name}/${local.container_name}"
```

Comment:
I still don't get it 😅 🤷🏽

Reply:
It's an example with available variables 🙂 
Thanks for reviewing!

For example, I want to create a CW log group with the `/aws/ecs/${local.name}/service/${local.container_name}` pattern and it was not possible in current implementation.
It was a possibility to create it additionally and pass through the variable, but I saw the same functionality for IAM roles and decided to make this small improvement to manage all required resources for ECS service logging inside the module.